doc/tutorials/PlutusTx.hs

```diff
@@ -0,0 +1,182 @@
+-- BLOCK1
+-- Necessary language extensions for the Plutus Tx compiler to work.
+{-# LANGUAGE TemplateHaskell     #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE NoImplicitPrelude   #-}
+module Tutorial.PlutusTx where
+
+import qualified Language.PlutusCore.Universe as PLC
+-- Main Plutus Tx module.
+import Language.PlutusTx
+-- Additional support for lifting.
+import Language.PlutusTx.Lift
+-- Builtin functions.
+import Language.PlutusTx.Builtins
+-- The Plutus Tx Prelude, discussed further below.
+import Language.PlutusTx.Prelude
+
+-- Setup for doctest examples.
+
+-- $setup
```

Comment:
The setup code doesn't need to be in the generated documentation, maybe just put it in a separate BLOCK